router/router.py

```diff
@@ -128,6 +128,8 @@ def is_test(p):
             # in case it is not present.
             if '/unit/' in p:
                 return True
+            if '/androidTest/' in p:
```

Comment:
Please combine this with the previous `/unit/` check, and update the comment on line 127 as well. thanks!

Reply:
Thank you, I updated this.